contracts/src/Teleporter/ITeleporterMessenger.sol

```diff
@@ -204,4 +204,17 @@ interface ITeleporterMessenger {
         bytes32 destinationChainID,
         uint256 messageID
     ) external view returns (address feeAsset, uint256 feeAmount);
+
+    /**
+     * @dev Gets the number of receipts that have been sent to the given destination chain ID.
+     */
+    function getReceiptQueueSize(bytes32 chainID) external view returns (uint256 size);
+
+    /**
+     * @dev Gets the receipt at the given index in the queue for the given chain ID.
+     */
+    function getReceiptAtIndex(
```

Comment:
can you add a `@param` comment about how `index` is compared to the front of the queue, and possibly example that `index=0` returns next receipts to be sent

Reply:
added `@param`